### ISSUE TITLE
Forever waiting for proxy

### ISSUE THREAD

**Issue by neskk** [drallieiv/KinanCity#58]
```
06-21 01:25:37 [Misty] PTC softban, put that IP on hold.
06-21 01:25:37 [Misty] Will retry
06-21 01:26:07 [Misty] Still no proxy slots available. Keep waiting
... (always the same message)
06-21 11:09:07 [Misty] Still no proxy slots available. Keep waiting
```

This error also happens when AccountWebServiceValidation fails. Don't know why or how but it enters a state where it waits forever for an available proxy.

If I terminate the App and restart the process it resumes it's operation no problem so I guess it wasn't really an IP ban or that it wasn't necessary to wait all those hours.

**Comment by drallieiv** [comment]
I would need the startup command that was used and details about the config file and either the full log or theses parts 
The log when the proxy was benched
Logs of the recycler running

Are you sure you are running with the last version, please add it in the issue description

**Comment by drallieiv** [comment]
I found an issue with 503 errors and slot management (again) where slots were not reset when getting a 503.  Is was patched in c43a709052a520250990ce8334cb84b32b0a8ba9
That should fix it.
